pages/more_examples.php

```diff
@@ -200,8 +200,7 @@ function show_errors($errors = array()) {
          $output = $moduls[$module_key]['output'];
 
          $mi = rex_sql::factory();
-         $mi->debugsql = 0;
-         $mi->setTable('rex_module');
```

Comment:
@Koala  Ich glaube setTable() darf hier nicht entfernt werden da ja in Zeile 202 eine neue Instanz von rex_sql erstellt wird.

Reply:
Oh, verd... ja, das war eine Zeile zu viel